### ISSUE TITLE
[Bug]: MineTogether Crash

### ISSUE THREAD

**Issue by DarkbrineOG** [FTBTeam/FTB-Modpack-Issues#1604]
### Modpack

FTB Stoneblock 3

### Modpack version

1.0.1

### Log Files

[crash-2022-11-03_12.26.59-client.txt](https://github.com/FTBTeam/FTB-Modpack-Issues/files/9931156/crash-2022-11-03_12.26.59-client.txt)

### Describe the bug

When I disabled my MineTogether (didn't do it on purpose), it seems as if I leave the chat without typing anything (and I'm not sure if it does the same after you type) it ends up crashing the Minecraft Client altogether.

### Steps to reproduce

1) Make sure your on a Stoneblock 3 world from the FTB Launcher.
2) Open up the chat and find the MineTogether prompt (it'll say connect online with the number of users playing or Don't Ask This Again [I think])
3) Click on the Button that says **Don't Ask This Again**
4) Crash will either follow on this point or if that doesn't happen..
5) Close the chat and open it again, crash should happen

### Expected behaviour

Chat should just close down normally without a crash.

### Screenshots

![image](https://user-images.githubusercontent.com/107524841/199777731-6a598e2c-52e8-4f31-93f4-7152126c11a3.png)

**Comment by CubeStick** [comment]
Can confirm, same happens to me.

**Comment by OneComputerGuy** [comment]
Can confirm, also happens if you click on the chat window after dismissing the message, here's the stacktrace:

```log
[03Nov2022 12:21:20.996] [pool-23-thread-1/INFO] [net.creeperhost.minetogether.lib.chat.irc.pircbotx.PircBotClient/]: #superspecialpirateclub: User#8B485 | string early game?
[03Nov2022 12:21:22.668] [pool-23-thread-1/INFO] [net.creeperhost.minetogether.lib.chat.irc.pircbotx.PircBotClient/]: #superspecialpirateclub: User#1BC11 | industral seeds
[03Nov2022 12:21:24.330] [pool-23-thread-1/INFO] [net.creeperhost.minetogether.lib.chat.irc.pircbotx.PircBotClient/]: #superspecialpirateclub: User#5D556 | User#8B485 hemp seeds, 3 hemp=1 string
[03Nov2022 12:21:33.554] [pool-23-thread-1/INFO] [net.creeperhost.minetogether.lib.chat.irc.pircbotx.PircBotClient/]: #superspecialpirateclub: User#1B12E | why is there a bee in the stone wall ? glowing bee
[03Nov2022 12:21:34.749] [pool-23-thread-1/INFO] [net.creeperhost.minetogether.lib.chat.irc.pircbotx.PircBotClient/]: #superspecialpirateclub: User#C9296 | is that actually a food?
[03Nov2022 12:21:37.118] [Render thread/INFO] [net.creeperhost.minetogether.lib.chat.irc.pircbotx.PircBotClient/]: Stopping MineTogether IRCClient.
[03Nov2022 12:21:37.968] [pool-23-thread-1/INFO] [net.creeperhost.minetogether.lib.chat.irc.pircbotx.PircBotClient/]: #superspecialpirateclub: User#81BF8 | I went with as short of a name as I could RCFC went a different route.
[03Nov2022 12:21:38.883] [pool-23-thread-1/INFO] [net.creeperhost.minetogether.lib.chat.irc.pircbotx.PircBotClient/]: Disconnected from MineTogether IRC.
[03Nov2022 12:21:38.883] [MineTogether IRC Client/INFO] [net.creeperhost.minetogether.lib.chat.irc.pircbotx.PircBotClient/]: Exiting Pircbotx MineTogether thread.
[03Nov2022 12:21:42.035] [Render thread/INFO] [configured/]: Unloading synced configs from server
[03Nov2022 12:21:42.035] [Render thread/INFO] [FluxNetworks/]: Released client Flux Networks cache
[03Nov2022 12:21:46.996] [Render thread/ERROR] [net.minecraft.client.Minecraft/FATAL]: Reported exception thrown!
net.minecraft.ReportedException: mouseClicked event handler
	at net.minecraft.client.gui.screens.Screen.m_96579_(Screen.java:535) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.MouseHandler.m_91530_(MouseHandler.java:85) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.MouseHandler.m_168091_(MouseHandler.java:185) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.util.thread.BlockableEventLoop.execute(BlockableEventLoop.java:103) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.MouseHandler.m_91565_(MouseHandler.java:184) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at org.lwjgl.glfw.GLFWMouseButtonCallbackI.callback(GLFWMouseButtonCallbackI.java:36) ~[lwjgl-glfw-3.2.2.jar%2348!/:build 10]
	at org.lwjgl.system.JNI.invokeV(Native Method) ~[lwjgl-3.2.2.jar%2344!/:build 10]
	at org.lwjgl.glfw.GLFW.glfwPollEvents(GLFW.java:3101) ~[lwjgl-glfw-3.2.2.jar%2348!/:build 10]
	at com.mojang.blaze3d.systems.RenderSystem.m_69495_(RenderSystem.java:204) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at com.mojang.blaze3d.platform.Window.m_85435_(Window.java:333) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.Minecraft.m_91383_(Minecraft.java:1068) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:665) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.main.Main.main(Main.java:205) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31) ~[fmlloader-1.18.2-40.1.84.jar%2317!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?]
Caused by: java.lang.NullPointerException: Cannot invoke "net.creeperhost.minetogether.polylib.gui.DropdownButton.m_6375_(double, double, int)" because "this.dropdownButton" is null
	at net.minecraft.client.gui.screens.ChatScreen.handler$bab000$onMouseClicked(ChatScreen.java:655) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.gui.screens.ChatScreen.m_6375_(ChatScreen.java) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.MouseHandler.m_168084_(MouseHandler.java:88) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.gui.screens.Screen.m_96579_(Screen.java:528) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	... 25 more
[03Nov2022 12:21:47.003] [Render thread/FATAL] [net.minecraftforge.common.ForgeMod/]: Preparing crash report with UUID d2b637a3-77e7-4271-a916-251c97ebba90
[03Nov2022 12:21:47.006] [Render thread/FATAL] [net.minecraftforge.common.ForgeMod/]: Preparing crash report with UUID e3fa686c-d7d5-4d75-9b7d-729a48dc542c
[03Nov2022 12:21:47.011] [Render thread/ERROR] [net.minecraft.client.main.Main/]: Unhandled game exception
java.lang.NullPointerException: Cannot invoke "org.apache.http.client.methods.HttpPost.getURI()" because "this.request" is null
	at dev.ftb.mods.ftbauxilium.ApiManager$PostRequest.post(ApiManager.java:208) ~[ftbauxilium-forge-1802.1.7-build.31-forge.jar%23135!/:?]
	at dev.ftb.mods.ftbauxilium.ApiManager.sendCrashReport(ApiManager.java:96) ~[ftbauxilium-forge-1802.1.7-build.31-forge.jar%23135!/:?]
	at net.minecraft.CrashReport.handler$zma000$saveToFile(CrashReport.java:522) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.CrashReport.m_127512_(CrashReport.java) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.Minecraft.m_91332_(Minecraft.java:781) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.Minecraft.m_91374_(Minecraft.java:685) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at net.minecraft.client.main.Main.main(Main.java:205) ~[client-1.18.2-20220404.173914-srg.jar%23263!/:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:?]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:?]
	at net.minecraftforge.fml.loading.targets.CommonClientLaunchHandler.lambda$launchService$0(CommonClientLaunchHandler.java:31) ~[fmlloader-1.18.2-40.1.84.jar%2317!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:53) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:71) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:106) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:77) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23) [modlauncher-9.1.3.jar%235!/:?]
	at cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149) [bootstraplauncher-1.0.0.jar:?]
[03Nov2022 12:21:47.012] [Render thread/INFO] [net.minecraft.client.Minecraft/]: Stopping!
[03Nov2022 12:21:47.013] [Render thread/INFO] [FluxNetworks/]: Released client Flux Networks cache
[03Nov2022 12:21:47.061] [Render thread/INFO] [FTB Chunks/]: Shutting down map thread
```

**Comment by Bigwooly** [comment]
I am getting this same error.  Has anyone found a work around to not have your game crash everytime you open chat after this?

**Comment by vanharen07** [comment]
> I am getting this same error. Has anyone found a work around to not have your game crash everytime you open chat after this?

disabling/removing the minetogether mod seems to not make it crash anymore.

**Comment by OneComputerGuy** [comment]
One workaround is to disable MineTogether from the mod list. On the FTB app open the modpack > click the three dots next to the "Play" button > Open > Mods folder and then changing the extension of the mod to something else

**Comment by Bigwooly** [comment]
> One workaround is to disable MineTogether from the mod list. On the FTB app open the modpack > click the three dots next to the "Play" button > Open > Mods folder and then changing the extension of the mod to something else

Thanks!

**Comment by Raidobw2** [comment]
Fixed in SB3 v1.0.2, which is currently available
